community-containers/nextcloud-exporter/readme.md

```diff
@@ -0,0 +1,43 @@
+## Prometheus Nextcloud Exporter
+
+A Prometheus exporter that collects metrics from your Nextcloud instance for monitoring and alerting.
+
+### What it does
+- Monitors user activity, file counts, and storage usage
+- Tracks system health, database size, and app statistics  
+- Exposes metrics on port `9205` for Prometheus to scrape
+- Uses Nextcloud's serverinfo API with secure token authentication
+
```

Comment:
I guess we should mention that the container should only be run in home networks because the service is not secured via token from the outside and potentially exposes private data, right?